Add more pins for Open Door pin

matjack1 opened the pull request:
Not sure if there was a reason behind limiting the pin options? For us would be useful to be able to pick the others as well, is it a problem?

matjack1 committed:
Add more pins for Open Door pin

omersiar commented:
Initial thought was SPI already occupied the said pins, that was the only option.  Thanks

matjack1 commented:
But @omersiar on my ESP8266 the SPI pins are mostly different from the ones I've added, have a look at this picture. Only GPIO-15 (between the ones I've added) is for SPI SS, which is also the only SPI pin that can be changed from the config, isn't it?

![wemos_d1_mini_pro_pinout](https://user-images.githubusercontent.com/65402/112461071-df23b180-8d5f-11eb-9a6c-f63c7837adbe.png)

I'm fine removing the SPI pins in general, but they are others, and we should get rid of them also in the other selects? If you want I can help getting everything more consistent.
